### DISCUSSION TITLE
Is it normal that several categories and formats can be selected?

### DISCUSSION THREAD

**Discussion by philippart-s** [conference-hall/conference-hall#274] [Q&A] [answered]
Hi,
It seems that, since the new version deployed, users cas set several formats and categories for their submissions.
Is it a bug or a new feature?

If it's a new feature can we have an option do disable it?

Cheers ;)

**Comment by bpetetot** [comment]
Hello,
Indeed it's a new feature, I agree organizers should be able to change this behaviour in the event settings.
We will prioritize it to have it quickly.

**Comment by bpetetot** [comment, marked as answer]
You can now enable/disable "formats and categories multiple selection" in your event settings 😉

By default, it is disabled (like the previous version of Conference Hall)

![image](https://github.com/user-attachments/assets/e7ce810c-d2d2-4be2-9dbd-07c5e3534e19)

**Comment by philippart-s** [comment]
Thanks a lot 💪
